src/features/dashboard/components/ApiTokenTable/table.token.cell.tsx

```diff
@@ -9,6 +9,10 @@ const ApiTokenCell = ({ cell }: React.PropsWithChildren<CellProps<TTokenType, st
   const has_admin_scope = cell.row?.original?.scopes?.includes('admin');
   const token = cell.value;
 
+  useEffect(() => {
+    setIsHidingToken(true);
+  }, [cell.value]);
```

Comment:
You do not need any value in your dependency array. You're not using cell.value inside your useEffect either.

```
useEffect(() => {
    setIsHidingToken(true);
 }, []);
```

I think this should work fine, since when you switch accounts, you rerender the component anyways.

Reply:
This will not work as the state change is dependant on the cell.value and not the component rerender

Reply:
You're right, just one nitpick though,

```
useEffect(() => {
    setIsHidingToken(true);
 }, [token]);
```

You can simply use the token variable in your dependency